package.json

```diff
@@ -64,7 +64,7 @@
     "react-helmet-async": "^1.3.0",
     "react-router-dom": "^6.15.0",
     "split2": "^4.2.0",
-    "undici": "^5.23.0",
+    "undici": "^5.24.0",
```

Comment:
@juliangruber do we still need `undici`? Can we use the built-in `fetch` implementation provided by recent Node.js versions instead? I don't know if our Electron version provides built-in `fetch` though.

Reply:
<img width="1192" alt="Screenshot 2023-09-12 at 07 37 34" src="https://github.com/filecoin-station/desktop/assets/10247/de33e718-aba2-483d-a5c1-b7cd25343cd8">

Looks like we can remove it 👍

Reply:
IIUC, your screenshot is showing `fetch` in action in the UI, which is running inside the Chromium side (`renderer`)

We need to check if we can call `fetch` on the Node.js side (`main`).

Reply:
Got it!

```
FETCH [AsyncFunction: fetch]
```

It is supported in the main process too :)